packages/@expo/config-plugins/src/android/Updates.ts

```diff
@@ -151,6 +153,15 @@ export async function setUpdatesConfigAsync(
     );
   }
 
+  const useEmbeddedUpdate = getUpdatesUseEmbeddedUpdate(config);
+  if (useEmbeddedUpdate === false) {
+    addMetaDataItemToMainApplication(
+      mainApplication,
+      Config.UPDATES_HAS_EMBEDDED_UPDATE,
+      String(useEmbeddedUpdate)
+    );
+  }
```

Comment:
do we need an else branch here with a call to `removeMetaDataItemFromMainApplication`? I think we probably should so that if the user changes it to true we remove the key from the manifest.